server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyFutureServerOptions.scala

```diff
@@ -3,49 +3,48 @@ package sttp.tapir.server.netty
 import com.typesafe.scalalogging.Logger
 import sttp.monad.{FutureMonad, MonadError}
 import sttp.tapir.model.ServerRequest
-import sttp.tapir.server.interceptor.log.{DefaultServerLog, ServerLog, ServerLogInterceptor}
+import sttp.tapir.server.interceptor.log.{DefaultServerLog, ServerLog}
 import sttp.tapir.server.interceptor.{CustomiseInterceptors, Interceptor}
 import sttp.tapir.{Defaults, TapirFile}
 
 import scala.concurrent.{Future, blocking}
 
 case class NettyFutureServerOptions(
-    host: String,
-    port: Int,
     interceptors: List[Interceptor[Future]],
     createFile: ServerRequest => Future[TapirFile],
     deleteFile: TapirFile => Future[Unit],
     nettyOptions: NettyOptions
 ) {
-  def host(s: String): NettyFutureServerOptions = copy(host = s)
-  def port(p: Int): NettyFutureServerOptions = copy(port = p)
-  def randomPort: NettyFutureServerOptions = port(0)
```

Comment:
what happened to `.randomPort`? Maybe we could add it to the top-level classes

Reply:
It was moved to NettyOptionsBuilder. 
An example is available under: 
`NettyFutureTestServerInterpreter:22`

If that option is frequently used I could add it to `NettyCatsServer` and also for `NettyFutureServer` but personally I think that this builder is enough.

Reply:
ok :)